index.js

```diff
@@ -1,18 +1,11 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler");
+const session = require("express-session"),
+  cors = require("cors"),
+  errorhandler = require("errorhandler"),
+  app = require("./app");
 
 const isProduction = process.env.NODE_ENV === "production";
 
-// Create global app object
-const app = express();
+require('dotenv').config();
```

Comment:
I thought this pr had to do with just test integration, why then am I seeing unrelated changes ? that means you started working off someone's branch. Your changes have to be yours and yours alone please fix this by rebasing staging